plugins/flytekit-openai/setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+
+PLUGIN_NAME = "chatgpt"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=1.10.7", "openai>=1.12.0", "flyteidl>=1.11.0b0"]
```

Comment:
Should we cut a beta release of flytekit?
Since SycnAgentBase needs to be ">1.10.7" version.

Reply:
cc @pingsutw

Reply:
not necessary now. you can use flytekit>1.0.7. CI will always install flytekit from master branch

Reply:
fixed, thx